spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Vm do
+  let(:ems) do
+    uid_ems  = "473f85b4-c4ba-4425-b495-d26c77365c91"
+    auth_key = Rails.application.secrets.ibmcvs.try(:[], :api_key) || "IBMCVS_API_KEY"
+    FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud, :uid_ems => uid_ems, :provider_region => "us-south").tap do |ems|
+      ems.authentications << FactoryBot.create(:authentication, :auth_key => auth_key)
+    end
+  end
+  let(:vm) { FactoryBot.create(:vm_ibm_cloud_power_virtual_servers, :ext_management_system => ems) }
+
+  context "is_available?" do
+    let(:power_state_on)        { "running" }
```

Comment:
The two remaining spec fails are caused by the `:power_state_on` definition. Right now a raw_power_state of `ACTIVE` results in an "on" state, and anything else results in "off":
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/vm.rb#L28

Try setting:
```
    let(:power_state_on)        { "ACTIVE" }
    let(:power_state_suspended) { "NOT_ACTIVE" }
```